docs/changelog/1799.misc.rst

```diff
@@ -0,0 +1 @@
+Enable building tox with ``setuptools_scm`` 4 and 5 by :user:hroncok
```

Comment:
@hroncok you missed the backticks here ^

Reply:
Where?

Reply:
At your user name 👍🏻

Reply:
Oh, my. Sorry. https://github.com/tox-dev/tox/pull/1851